generator/Makefile

```diff
@@ -35,6 +35,8 @@ APC_URL           := https://download.schneider-electric.com/files?p_File_Name=p
 ARISTA_URL        := https://www.arista.com/assets/data/docs/MIBS
 CISCO_URL         := https://raw.githubusercontent.com/cisco/cisco-mibs/f55dc443daff58dfc86a764047ded2248bb94e12/v2
 DELL_URL          := https://dl.dell.com/FOLDER11196144M/1/Dell-OM-MIBS-11010_A00.zip
+DLINK_URL         := https://github.com/pgmillon/observium/raw/refs/heads/master/mibs/d-link/
```

Comment:
We generally prefer authoritative sources for MIBS, rather than mirrors created by other projects like Observium.

Reply:
agreed, found better one for dlink.

Reply:
unfortunately, I couldn't find frogfoot mib on the UBNT site.

Reply:
discussed this with @RichiH during FOSDEM and we think it is defensible to use this source in that case